spec/models/bishop_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Bishop, type: :model do
+  describe '#move_legal?' do
+    let(:bishop){Bishop.create(row: 3, col: 3)}
+
+    it "is legal" do

```

Comment:
A better description would be: `it "allows legal moves"`
Same thing for the second test.